core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -226,7 +226,7 @@ public static CarbonTable buildFromTablePath(String tableName, String tablePath,
     } else {
       // Infer the schema from the Carbondata file.
       TableInfo tableInfoInfer =
-          SchemaReader.inferSchema(AbsoluteTableIdentifier.from(tablePath, "null", "null"), false);
+          CarbonUtil.inferDummySchema(tablePath, "null", "null");
```

Comment:
Change the name of the method to `buildDummyTable(tableName, tablePath)

Reply:
OK. Changed